src/com/daimler/pipeliner/BasePipeline.groovy

```diff
@@ -774,8 +823,8 @@ abstract class BasePipeline implements Serializable {
     /**
      * Generates a job name based on the parallel keys and values.
      * Eg for parallel field "targets":
-     * Legacy: "target: qemu-x86-64_nogfx:core-image-minimal"
-     * New: "qemu-x86-64_nogfx:core-image-minimal"
+     * Legacy: "target: jetson-tx2_nogfx:core-image-mbient-dev"
```

Comment:
I don't think having `mbient` in the example here is appropriate because it's a internal thing, it should be more generic.